src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -37,58 +34,30 @@ const GovernanceSection = () => {
       <AccordionDetails sx={({ spacing }) => ({ padding: `0 ${spacing(3)}` })}>
         {claimingApp || fetchingSafeClaimingApp ? (
           <WidgetBody>
-            <Grid spacing={3} container>
-              <Grid item xs={12} md={6} lg={8}>
-                <Card sx={{ height: '300px' }}>
-                  {claimingApp ? (
-                    <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
-                      <AppFrame
-                        appUrl={`${claimingApp.url}${SNAPSHOT_WIDGET_ID}`}
-                        allowedFeaturesList={getAllowedFeaturesList(claimingApp.url)}
-                        isQueueBarDisabled
-                      />
-                    </SafeAppsErrorBoundary>
-                  ) : (
-                    <Box
-                      sx={{ height: '300px' }}
-                      display="flex"
-                      alignItems="center"
-                      justifyContent="center"
-                      textAlign="center"
-                    >
-                      <Typography variant="h1" color="text.secondary">
-                        Loading Snapshot...
-                      </Typography>
-                    </Box>
-                  )}
-                </Card>
-              </Grid>
-              <Grid item xs={12} md={6} lg={4}>
-                <Card sx={{ height: '300px' }}>
-                  {claimingApp ? (
-                    <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
-                      <AppFrame
-                        appUrl={`${claimingApp.url}${CLAIMING_WIDGET_ID}`}
-                        allowedFeaturesList={getAllowedFeaturesList(claimingApp.url)}
-                        isQueueBarDisabled
-                      />
-                    </SafeAppsErrorBoundary>
-                  ) : (
-                    <Box
-                      sx={{ height: '300px' }}
-                      display="flex"
-                      alignItems="center"
-                      justifyContent="center"
-                      textAlign="center"
-                    >
-                      <Typography variant="h1" color="text.secondary">
-                        Loading Claiming app...
-                      </Typography>
-                    </Box>
-                  )}
-                </Card>
-              </Grid>
-            </Grid>
+            <Card className={css.widgetWrapper}>
+              {claimingApp ? (
+                <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
```

Comment:
This widget should probably have a different fallback component?

Reply:
Better to involve Design on this? A variant of `<SafeAppsLoadError>` without the button to go back to the Safe Apps list would also do it. WDYT?

Reply:
Could be something simple, like the Loading message but saying "Couldn't load governance widgets". But yeah, let's ask the designer.

Reply:
@liliiaorlenko in case the widgets run into a problem loading can we display
<img width="907" alt="Screenshot 2022-11-24 at 15 48 21" src="https://user-images.githubusercontent.com/32431609/203813177-243b2e8a-2fa4-45c5-92cd-c3dafa7b43d7.png">

or would you suggest other design?

Reply:
Ill create a placeholder for it

Reply:
https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9/Governance-visibility-widgets?node-id=405%3A1857&t=fPVkONyGNzBoyedH-1

Reply:
Added the placeholders 
<img width="867" alt="Screenshot 2022-11-25 at 11 32 19" src="https://user-images.githubusercontent.com/32431609/203966278-4ba4734a-16e5-4ca7-ac46-25aa7d7b5698.png">

Reply:
These should be singular, e.g. "widget".